bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -2,6 +2,7 @@ import Foundation
 
 private func dummy() {
     _ = NSLocalizedString("'%s' is not fully-supported", comment: "translators: Missing block alert title. %s: The localized block name")
+    _ = NSLocalizedString("%1$s (%2$s)", comment: "translators: %1$s: Select control font size name e.g. Small, %2$s: Select control font size e.g. 12px")
```

Comment:
- Addition
- [Code reference](https://github.com/WordPress/gutenberg/blob/e393d55cacc030f43b7f64104a8ff1aa2b179a62/packages/components/src/font-size-picker/index.native.js#L75)

Looks like we missed this string, most likely this was caused due to the parsing issues when extracting the strings.